pycds/alembic/versions/a59d64cf16ca_add_hx_tkg_to_main_metadata_tables.py

```diff
@@ -45,19 +45,19 @@ def upgrade():
     # the history table is populated can find the functions that they call.
     op.get_bind().execute(f"SET search_path TO {schema_name}, public")
 
-    for table_name, primary_key_name, foreign_keys, extra_indexes in table_info:
+    for table_name, primary_key_name, foreign_tables, extra_indexes in table_info:
         # Primary table
         add_history_cols_to_primary(table_name)
         create_primary_table_triggers(table_name)
 
         # History table
-        create_history_table(table_name, foreign_keys)
+        create_history_table(table_name, foreign_tables)
+        populate_history_table(table_name, primary_key_name, foreign_tables)
```

Comment:
it looks like this function takes over the previous responsibilities of `update_obs_raw_history_FKs` via the foreign table ctes within. Suggest removing `update_obs_raw_history_FKs` from the imports and utils entirely if it is no longer needed